.github/workflows/dev_envs.yml

```diff
@@ -16,9 +16,14 @@ jobs:
         extra_nix_config: |
           access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
 
+    - uses: cachix/cachix-action@v10
+      with:
+        name: sourmash-bio
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+
     - run: nix run .# -- --version
 
-    - run: nix-shell --command "tox -e py39"
+    - run: nix-shell --command "tox -e py310"
```

Comment:
run with the same Python version for the nix package, so we don't need to download/build two Python versions